spec/fabricators/post_fabricator.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 Fabricator(:post) do
-  user
+  user { |attrs| attrs[:user] || Fabricate(:user, refresh_auto_groups: true) }
```

Comment:
I did this in one of my PRs as well. 👍 Curious. Is the `attrs[:user]` condition needed? I would assume it only uses the block if no value provided, but I'm not familiar enough to say for sure.

Reply:
Oh yeah I think you are probably right about the `attrs` :+1: